DataConnectors/Gitlab/azuredeploy.json

```
@@ -0,0 +1,317 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+      "PlaybookName": {
+            "defaultValue": "Gitlab-Sentinel",
+            "type": "string"
+        },
+        "PersonalAccessToken": {
+            "type": "securestring",
+            "defaultValue": "<Personal Access Token>",
+            "metadata": {
+                "description": "The personal access token for gitlab instance with read access."
+            }
+        },
+        "ProjectID": {
+            "type": "string",
+            "defaultValue": "<ProjectID>",
+            "metadata": {
+                "description": "The project id for gitlab projects to be audited."
```

```suggestion
                "description": "The Project IDs for GitLab projects to be audited."
```

The description does current reflect that multiple Project IDs should be accepted.  If you do not support multiple Project IDs then

```suggestion
                "description": "The Project ID for the GitLab project to be audited."
```